packages/nextjs/hooks/scaffold-eth/useScaffoldContract.ts

```diff
@@ -8,23 +8,31 @@ import { Contract, ContractName } from "~~/utils/scaffold-eth/contract";
  * Gets a deployed contract by contract name and returns a contract instance
  * @param config - The config settings
  * @param config.contractName - Deployed contract name
+ * @param config.proxyContractName - Deployed proxy contract name if you wish to use an address other than the one associated with the contractName (optional)
```

Comment:
I think it is better to have something like "(Only for proxy contracts)" at the beginning of the comment, so it's clear at the very start that you don't need to worry about these parameters if you are not working with proxies.